geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/SpatioTemporalTable.scala

```diff
@@ -86,4 +90,20 @@ object SpatioTemporalTable extends GeoMesaTable with Logging {
     bd.delete()
     bd.close()
   }
+
+
+  override def configureTable(sft: SimpleFeatureType, tableName: String, tableOps: TableOperations): Unit = {
+    // NOTE: since the ST table is likely going away, I'm not inclined to thread maxShards all the way through
+    // the call chain so I just set a default of 40 here
```

Comment:
I'm fine with this right now.  In order to codify routing extras info, how about widening configureTable to take a Map[String, String]?